examples/axes_grid1/demo_axes_rgb.py

```diff
@@ -58,22 +65,20 @@ def demo_rgb():
 def demo_rgb2():
     fig = plt.figure(2)
     ax = RGBAxes(fig, [0.1, 0.1, 0.8, 0.8], pad=0.0)
-    #fig.add_axes(ax)
-    #ax.add_RGB_to_figure()
+    # fig.add_axes(ax)
+    # ax.add_RGB_to_figure()
 
     r, g, b = get_rgb()
     kwargs = dict(origin="lower", interpolation="nearest")
     ax.imshow_rgb(r, g, b, **kwargs)
 
-    ax.RGB.set_xlim(0., 9.5)
+    ax.RGB.set_xlim(0., 10.6)
     ax.RGB.set_ylim(0.9, 10.6)
 
     for ax1 in [ax.RGB, ax.R, ax.G, ax.B]:
-        for sp1 in ax1.spines.values():
-            sp1.set_color("w")
-        for tick in ax1.xaxis.get_major_ticks() + ax1.yaxis.get_major_ticks():
-            tick.tick1line.set_mec("w")
-            tick.tick2line.set_mec("w")
+        ax1.tick_params(axis='both', direction='in')
+        ax1.xaxis.set_ticks(np.arange(0, 12, 2))
+        ax1.yaxis.set_ticks(np.arange(0, 12, 2))
```

Comment:
You can take the two tick-setters outside the loop and apply them to `ax.RGB` (or any one of the 4).  There is no telling how long it will take before the underlying bug is found and fixed, so this is a reasonable workaround in the interim.  In addition, once the bug is fixed, the automatically-located ticks will be based on the small `ax.B` so there will be fewer of them than you want.